src/ServicePulse.Host/app/js/views/configuration/configuration.html

```diff
@@ -17,14 +17,21 @@ <h1>Configuration</h1>
 
         <div class="row">
             <div class="col-sm-12">
+
+                <div class="alert alert-warning">
+                    <i class="fa fa-warning"></i> <strong>Warning:</strong> The list of endpoints below only contains endpoints with the hearbeats plug-in installed. Toggling heartbeat monitoring on/off won't toggle <a href="https://docs.particular.net/monitoring/metrics/in-servicepulse" target="_blank">performance monitoring</a> <i class="fa fa-external-link fake-link"></i>
+                </div>
+
                 <div class="row box box-no-click table-header">
                     <div class="col-xs-3 col-sm-2 col-lg-1 no-side-padding">
-                        <p>Monitoring</p>
+                        <p>Heartbeats monitoring</p>
```

Comment:
This title feels clunky.

What about just `Heartbeats`?
Or perhaps even dropping the title completely given that the description explains the list below?

Reply:
I'm good with making it just `Heartbeats`. We should keep those headers to leave no room for doubts about what those toggles are for.